projects/plugins/crm/src/js/components/automations/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { __ } from '@wordpress/i18n';
+
+import './styles.scss';
+
+export const AutomationsAdmin = () => {
```

Comment:
The file name and component name doesn't match.

Referencing it as "Admin" or "Page" is otherwise 👍 with me

Reply:
I switched the file name.